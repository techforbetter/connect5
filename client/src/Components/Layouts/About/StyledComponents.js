import styled from "styled-components";
import Header from "../../CommonComponents/Header/index";

const Container = styled.div`
  display: Grid;
  grid-template-columns: 1fr;
  grid-template-rows: 55px 250px 190px auto auto;
  grid-template-areas:
    "header"
    "summury"
    "mission"
    "about"
    "blockquote"

`;

const Headers = styled(Header)`
  grid-area: header;
`;

const Summury = styled.div`
  grid-area: summury;
  background: #dbeeed;
`;

const H2 = styled.h2`
  display: block;
  font-size: 1.5rem;
  font-family: cursive,serif;
  font-weight: 700;
  line-height: 1.2;
  margin-bottom: 22px;
  text-align: center;
  color: var(--main-heading);
`;

const H3 = styled(H2)`
  display: block;
  font-size: 1.4rem;
  font-weight: 400;
  text-align: center;
  color: #393939;
`;

const P = styled.p`
  display: block;
  line-height: 1.4;
  text-align: center;
  margin: 0px 10px;
  color: #6a6a6a;
`;

const Mission = styled.div`
  grid-area: mission;
`;

const About = styled.div`
  grid-area: about;
  display: flex;
  flex-direction: column;
  background: #fff8fd;
`;

const Question = styled.div`
 display: flex;
 flex-direction: column;
 width: 90%;
 margin: 0 auto;
 margin-bottom: 20px;
 color: #6a6a6a;
 padding: 0 10px;
 border: 2px solid #fff;
 border-radius: 9px;
 overflow: hidden;
 box-sizing: border-box;
 -webkit-box-shadow: 0px 0px 2px 0px rgba(0, 0, 0, 0.31);
 -moz-box-shadow: 0px 0px 2px 0px rgba(0, 0, 0, 0.31);
 box-shadow: 0px 0px 2px 0px rgba(0, 0, 0, 0.31);
`;

const Paragraph = styled.p`
 margin: 0 0 30px;
 line-height: 1.4;
 font-weight: 400;
 color: #6a6a6a;
`;

const Ul = styled.li`
 padding-left: 10px;
 margin-bottom: 20px;
 list-style: none;
 list-style-position: outside;
`;

const Li = styled.li`
  margin-bottom: 25px;
  padding-left: 10px;
  display: list-item;
  text-align: left;
  list-style: none;
  font-size: 16px;
`;


const Blockquote = styled.div`
  display: block;
  grid-area: blockquote;
  background-color: #fbeef7;
  border: none;
  color: #e73e64;
  font-family: Merriweather,serif;
  font-size: 20px;
  font-weight: 400;
  position: relative;
  margin: 0 0 20px;
  width: 100%;
  padding: 20px;
`;

const Block = styled.blockquote`
  margin-top: 30px;
  width: 100%;
  &:before {
    content: "\2706";
    font-size: 30px;
  };
   &:after { content: '\2192'; }

`;

const Quote = styled.p`
  margin-bottom: 20px;
  margin: 40 0 20px;
  line-height: 1.4;
  font-size: 18px;
`;

const Speaker = styled.p`
  color: #3e3327;
  font-size: 16px;
  margin: 0 0 30px;
  line-height: 1.4;
`;

export {
  Container,
  Speaker,
  Quote,
  Block,
  Blockquote,
  Paragraph,
  Li,
  Ul,
  Question,
  About,
  Mission,
  P,
  Summury,
  Headers,
  H2,
  H3,
};
