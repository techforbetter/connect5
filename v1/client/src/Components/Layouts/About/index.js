import React from "react";
import {
  Container,
  Speaker,
  Quote,
  Block,
  Blockquote,
  Li,
  Ul,
  Question,
  About,
  Mission,
  Paragraph,
  P,
  Summury,
  Headers,
  H2,
  H3,
} from "./StyledComponents";

const AboutUs = () => (
  <Container>
    <Headers />
    <Summury>
      <H2>Connect5</H2>
      <P>
        Connect 5 has been developed to give frontline staff the confidence to
        have more effective conversations with the public about their mental
        health and wellbeing
      </P>
    </Summury>
    <Mission>
      <H3>-Our Mission-</H3>
      <P>
        To provides participants with skills and competencies that build
        confidence in having conversations about mental health and wellbeing in
        their routine practice.
      </P>
    </Mission>
    <About>
      <H3>About</H3>
      <Question>
        <h3>What is Connect 5?</h3>
        <Paragraph>
          Connect 5 is focused on training staff to enable them to support their
          patients or clients to live well mentally, to better manage their
          mental health, increase their resilience and ultimately improve
          wellbeing.
        </Paragraph>
        <Paragraph>
          The techniques used by Connect 5 are underpinned by the Five Ways to
          Wellbeing and a cognitive behavioural model for understanding stress,
          distress and learning tools for self-management.
        </Paragraph>
        <Paragraph>
          Connect 5 training is tailored to the needs of your staff – and can
          take place for between a half-day and two and a half days depending on
          who it is for and how much support they can provide.
        </Paragraph>
      </Question>
      <Question>
        <h3>Why has it been developed?</h3>
        <Paragraph>
          With mental health emerging as a pressing priority, and services
          struggling to cope with the burden, promoting self-management is at
          the heart of current health and social care policy.
        </Paragraph>
        <Paragraph>
          There are currently many obstacles preventing people from accessing
          the help they need with mental health issues. Stigma associated with
          mental health means that those not working in mental health often
          don’t feel skilled or confident enough to speak about mental health
          issues.
        </Paragraph>
        <Paragraph>
          Connect 5 has been developed to help address these problems by
          releasing some of the core knowledge and skills held by specialist
          mental health services into the wider health and care workforce,
          enabling people to feel confident having conversations around health
          and wellbeing, and make more effective referrals in to mental health
          services.
        </Paragraph>
      </Question>
      <Question>
        <h3>
          How does Connect 5 fit with national health and care priorities?
        </h3>
        <Ul>
          <Li>
            Builds capacity and capability to support prevention and reduce
            demand, supporting the Five Year Forward View (FYFV) and
            specifically the FYFV for Mental Health workforce strategy.
          </Li>
          <Li>
            Delivers brief interventions for mental wellbeing as part of Making
            Every Contact Count (MECC), as agreed in the MECC national consensus
            statement.
          </Li>
          <Li>
            Promotes a self-care and self-management approach and increases
            mental health literacy.
          </Li>
          <Li>
            Champions the public’s health, wellbeing, independence and control,
            supporting the Care Act and Public Health priorities.
          </Li>
          <Li>
            Develops the role of the ‘wider workforce’ to embed prevention and
            public health across a broader system, in line with the Fit for the
            Future Public Health strategy and the Public
          </Li>
          <Li>Mental Health Leadership and Workforce Development Framework</Li>
        </Ul>
      </Question>
    </About>
    <Blockquote>
      <H3>What did people say about us?</H3>
      <Block>
        <Quote>
          Connect 5 helps to support the aspirations outlined in the PHE
          Prevention Concordat by facilitating universal responsibility for
          prevention and the promotion of mental wellbeing. It is applicable
          across the whole of the public sector workforce, employers and
          community groups everywhere.
        </Quote>
        <Speaker>
          Julie Daneshyar, Health and Wellbeing Programme Manager, Public Health
          England
        </Speaker>
      </Block>
    </Blockquote>
  </Container>
);

export default AboutUs;
