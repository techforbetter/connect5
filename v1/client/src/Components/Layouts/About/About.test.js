import React from "react";
import renderer from "react-test-renderer";
import { StaticRouter } from "react-router-dom";

import About from "./index";

test("About matches snapshot", () => {
  const context = {};
  const component = renderer.create(
    <StaticRouter context={context}>
      <About />
    </StaticRouter>,
  );
  expect(component).toMatchSnapshot();
});
