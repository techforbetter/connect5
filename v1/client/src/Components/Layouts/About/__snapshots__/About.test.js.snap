// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About matches snapshot 1`] = `
<div
  className="sc-gzVnrw gCpeHT"
>
  <div
    className="sc-bxivhb igcOnr"
  >
    <img
      alt="Logo"
      className="sc-bZQynM fSermK"
      src="connect5_logo_main.jpg"
    />
    <div
      className="sc-EHOje eLNQHg"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        <i
          className="fas fa-home sc-ifAKCX bmGznW"
        />
      </a>
    </div>
  </div>
  <div
    className="sc-dnqmqq enFpMa"
  >
    <h2
      className="sc-iwsKbI cAKfPL"
    >
      Connect5
    </h2>
    <p
      className="sc-gqjmRU jfgWIf"
    >
      Connect 5 has been developed to give frontline staff the confidence to have more effective conversations with the public about their mental health and wellbeing
    </p>
  </div>
  <div
    className="sc-VigVT gYOfdU"
  >
    <h2
      className="sc-iwsKbI sc-gZMcBi fmHcAu"
    >
      -Our Mission-
    </h2>
    <p
      className="sc-gqjmRU jfgWIf"
    >
      To provides participants with skills and competencies that build confidence in having conversations about mental health and wellbeing in their routine practice.
    </p>
  </div>
  <div
    className="sc-jTzLTM hEAqqe"
  >
    <h2
      className="sc-iwsKbI sc-gZMcBi fmHcAu"
    >
      About
    </h2>
    <div
      className="sc-fjdhpX bBIaYn"
    >
      <h3>
        What is Connect 5?
      </h3>
      <p
        className="sc-jzJRlG cSUhzZ"
      >
        Connect 5 is focused on training staff to enable them to support their patients or clients to live well mentally, to better manage their mental health, increase their resilience and ultimately improve wellbeing.
      </p>
      <p
        className="sc-jzJRlG cSUhzZ"
      >
        The techniques used by Connect 5 are underpinned by the Five Ways to Wellbeing and a cognitive behavioural model for understanding stress, distress and learning tools for self-management.
      </p>
      <p
        className="sc-jzJRlG cSUhzZ"
      >
        Connect 5 training is tailored to the needs of your staff – and can take place for between a half-day and two and a half days depending on who it is for and how much support they can provide.
      </p>
    </div>
    <div
      className="sc-fjdhpX bBIaYn"
    >
      <h3>
        Why has it been developed?
      </h3>
      <p
        className="sc-jzJRlG cSUhzZ"
      >
        With mental health emerging as a pressing priority, and services struggling to cope with the burden, promoting self-management is at the heart of current health and social care policy.
      </p>
      <p
        className="sc-jzJRlG cSUhzZ"
      >
        There are currently many obstacles preventing people from accessing the help they need with mental health issues. Stigma associated with mental health means that those not working in mental health often don’t feel skilled or confident enough to speak about mental health issues.
      </p>
      <p
        className="sc-jzJRlG cSUhzZ"
      >
        Connect 5 has been developed to help address these problems by releasing some of the core knowledge and skills held by specialist mental health services into the wider health and care workforce, enabling people to feel confident having conversations around health and wellbeing, and make more effective referrals in to mental health services.
      </p>
    </div>
    <div
      className="sc-fjdhpX bBIaYn"
    >
      <h3>
        How does Connect 5 fit with national health and care priorities?
      </h3>
      <li
        className="sc-cSHVUG jdvvgH"
      >
        <li
          className="sc-kAzzGY eXhUlk"
        >
          Builds capacity and capability to support prevention and reduce demand, supporting the Five Year Forward View (FYFV) and specifically the FYFV for Mental Health workforce strategy.
        </li>
        <li
          className="sc-kAzzGY eXhUlk"
        >
          Delivers brief interventions for mental wellbeing as part of Making Every Contact Count (MECC), as agreed in the MECC national consensus statement.
        </li>
        <li
          className="sc-kAzzGY eXhUlk"
        >
          Promotes a self-care and self-management approach and increases mental health literacy.
        </li>
        <li
          className="sc-kAzzGY eXhUlk"
        >
          Champions the public’s health, wellbeing, independence and control, supporting the Care Act and Public Health priorities.
        </li>
        <li
          className="sc-kAzzGY eXhUlk"
        >
          Develops the role of the ‘wider workforce’ to embed prevention and public health across a broader system, in line with the Fit for the Future Public Health strategy and the Public
        </li>
        <li
          className="sc-kAzzGY eXhUlk"
        >
          Mental Health Leadership and Workforce Development Framework
        </li>
      </li>
    </div>
  </div>
  <div
    className="sc-chPdSV bzBafG"
  >
    <h2
      className="sc-iwsKbI sc-gZMcBi fmHcAu"
    >
      What did people say about us?
    </h2>
    <blockquote
      className="sc-kgoBCf kVSTZj"
    >
      <p
        className="sc-kGXeez kyrwpq"
      >
        Connect 5 helps to support the aspirations outlined in the PHE Prevention Concordat by facilitating universal responsibility for prevention and the promotion of mental wellbeing. It is applicable across the whole of the public sector workforce, employers and community groups everywhere.
      </p>
      <p
        className="sc-kpOJdX cSLMdC"
      >
        Julie Daneshyar, Health and Wellbeing Programme Manager, Public Health England
      </p>
    </blockquote>
  </div>
</div>
`;
